src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.ts

```diff
@@ -64,27 +96,43 @@ export class CirclesEditorComponent implements OnDestroy {
         const circle = this.circlesManager.get(update.id);
         if (update.radiusPoint) {
           circle.movePoint(update.radiusPoint);
+          this.renderEditLabels(circle, update);
```

Comment:
why render labels logic is here and all other circle,point,outline is in editableCircle ?

Reply:
not related to the editable geometry - it is part of the visual features that the component handles